docs/third-party-developers/extensibility/README.md

```diff
@@ -32,6 +32,7 @@ See <https://www.npmjs.com/package/@woocommerce/dependency-extraction-webpack-pl
 | ----------------------------- | ------------------------------------------------------- |
 | [Actions](./hooks/actions.md) | Documentation covering action hooks on the server side. |
 | [Filters](./hooks/filters.md) | Documentation covering filter hooks on the server side. |
+| [Migrated Hooks](./hooks/migrated-hooks.md) | Documentation listing the migrated WooCommerce core hooks. |
```

Comment:
Nice catch! Thank you for updating the doc. Do you think we should keep the text consistent with the Actions and Filters documentation?

```suggestion
| [Migrated Hooks](./hooks/migrated-hooks.md) | Documentation covering the migrated WooCommerce core hooks. |
```